lib/util/git/url.ts

```diff
@@ -32,7 +32,7 @@ export function getHttpUrl(url: string, token?: string): string {
     }
   }
 
-  return parsedUrl.toString(protocol);
+  return new URL(parsedUrl.toString(protocol)).href;
```

Comment:
Can you update or add a test to verify the new behaviour?

Reply:
I added a new test to verify the extraction process.